DA-238: Implement pagination for projects view

JeannedArk opened the pull request:
The pagination view increases the performance of the projects view
tremendously because not all projects which belong to a user are
requested and loaded anymore, only 5 per page. Therefore all
related requests and views had to be adjusted.

stgrue commented:
When adding a new scheme, it does not show up in the list until you refresh the page. (It appears as though this bug is not present in the master branch.)

stgrue commented:
The performance is very good, have tested it with 25 projects with 250+ documents each.

JeannedArk committed:
DA-238: Implement pagination for projects view

The pagination view increases the performance of the projects view
tremendously because not all projects which belong to a user are
requested and loaded anymore, only 5 per page. Therefore all
related requests and views had to be adjusted.
